apisix/cli/ops.lua

```diff
@@ -728,14 +729,16 @@ local function start(env, ...)
     local pid = util.read_file(pid_path)
     pid = tonumber(pid)
     if pid then
-        local lsof_cmd = "lsof -p " .. pid
-        local res, err = util.execute_cmd(lsof_cmd)
-        if not (res and res == "") then
-            if not res then
-                print(err)
-            else
-                print("APISIX is running...")
-            end
+        local signone = 0
+        local errno_noproc = 3
+
+        local ok, err, errno = signal.kill(pid, signone)
```

Comment:
```suggestion
        local no_proc = 3

        local ok, err, code = signal.kill(pid, signone)
```

is better?

Reply:
I get

Reply:
`errno` -> `err_no` is better?

It seems common within APISIX to use underscores to separate words

Reply:
Did it.